scrapers/covid-19/govScrapers/getCanada.js

```diff
@@ -1,39 +1,31 @@
 const axios = require('axios');
-const cheerio = require('cheerio');
 const logger = require('../../../utils/logger');
-
-const columns = ['province', 'cases', 'deaths'];
+const csvUtils = require('../../../utils/csvUtils');
+const moment = require('moment');
 
 /**
- * Return object reflecting a row of data from Canadian government site
- * @param 	{number} 	_ 		Index getting passed when using .map()
- * @param 	{Object} 	row		The row to extract data from
- * @returns {Object}				Data for canadian province with entries for each column in @constant columns
+ * Return array of provinces that match today's date (initial csv is historical)
+ * @param 	{Object} 	csv		The row to extract data from
+ * @returns {Array}				Data for canadian province
  */
-const mapRows = (_, row) => {
-	const province = { updated: Date.now() };
-	cheerio(row).children('td').each((index, cell) => {
-		cell = cheerio.load(cell);
-		switch (index) {
-			case 0: {
-				province[columns[index]] = cell.text() === 'Canada' ? 'Total' : cell.text();
-				break;
-			}
-			default: {
-				province[columns[index]] = parseInt(cell.text().replace(/,/g, '')) || null;
-			}
-		}
-	});
-	return province;
+const filterByDate = (csv) => {
+	const date = moment().format('DD-MM-YYYY');
+	return csv.filter(row => row.date === date);
```

Comment:
I'd rather not install moment.js for this purpose. It's relatively easy to get today's date in the format of `dd-mm-yyyy`. I'll post a snippet below:
```js
const d = new Date();
return `${d.getDate().padStart(2, '0')}${d.getMonth()}${d.getYear()}`
```